bin/desi_proc

```diff
@@ -296,24 +314,30 @@ if args.batch:
         cmd = ' '.join(inparams)
         cmd = cmd.replace(' --batch', ' ').replace(' --nosubmit', ' ')
 
+        timingfile = f'{jobname}-timing-$SLURM_JOBID.json'
```

Comment:
Should this be using the arg.timingfile from the parser or at least checking for its existence before defining to default?

Reply:
Good point; I added a check to use `args.timingfile` here.

Default usage with `--batch` is to *not* specify `--timingfile`, and then let `desi_proc` pick a timing file name based upon the job name and slurm jobid; but if the user specifies `--batch` and a specific `--timingfile`, that will be used instead.

Reply:
Great. I think this is the best approach even if its not meant to be used by end users. It's either this or documenting the use cases, and this is simpler and more flexible.